### DISCUSSION TITLE
Lesson 6: Error Creating mainnet-fork?

### DISCUSSION THREAD

**Discussion by hai-yr** [smartcontractkit/full-blockchain-solidity-course-py#790] [Q&A] [answered]
I am trying to add the development mainnet-fork using the following command:

```
hai-yr-MacBook-Pro smartcontract-lottery % brownie networks add development mainnet-fork -dev cmd=ganache-cli host=http://127.0.0.1 fork=https://eth-mainnet.alchemyapi.io/v2/ALCHEMY_PROJECT_ID accounts=10 mnemonic=brownie port=8545
```
the response I see on terminal:
```
Brownie v1.17.2 - Python development framework for Ethereum

Usage: brownie networks <command> [<arguments> ...] [options]
```
and when I check networks I see this:
`─Ganache-CLI (Mainnet Fork)
  │ ├─id: mainnet-fork
  │ ├─cmd: ganache-cli
  │ ├─host: http://127.0.0.1
  │ └─timeout: 120
  │   ├─accounts: 10
  │   ├─evm_version: istanbul
  │   ├─fork: mainnet
  │   ├─gas_limit: 12000000
  │   ├─mnemonic: brownie
  │   └─port: 8545`

But when I run `networks --delete mainnet-fork -dev` I get `Usage: brownie networks <command> [<arguments> ...] [options]` again...  and can still see the "mainnet fork" shown above. Is this an error message? The cli is definitely not giving the success response that is shown in the tutorial.

**Comment by NathanPtrs** [comment, marked as answer]
I'm seeing a space in "mainnet-fork -dev"  Were you intending to do "mainnet-fork-dev" (no space)? The command that worked for me was:
brownie networks add development mainnet-fork cmd=ganache-cli host=http://127.0.0.1 fork=https://eth-mainnet.alchemyapi.io/v2/[MYKEY] accounts=10 mnemonic=brownie port=8545

**Reply by cromewar** [reply]
Yes @hai-yr have a space on "mainnet-fork -dev" so brownie is not recognizing it as a real command. You are awesome @NathanPtrs !!

**Reply by hai-yr** [reply]
thanks guys will give it a shot
